kotlinx-coroutines-core/common/src/AbstractCoroutine.kt

```diff
@@ -71,7 +71,7 @@ public abstract class AbstractCoroutine<in T>(
      */
     protected open fun onStart() {}
 
-    internal final override fun onStartInternal() {
+    final override fun onStartInternal() {
```

Comment:
What's the rationale of making a lot of API `public`, avoid name mangling and bridges?

Reply:
Yes. It is a minor improvement in size, but still.

Reply:
I've got rid of those tweaks. It makes no difference with obfuscation.